index.html

```diff
@@ -12,10 +12,56 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    <nav class="navbar">
```

Comment:
Along the lines of thinking of semantic html, the menu options are always at the top of the page in the Karma website - logically the `nav` element could very well fit within a `<header>` element.